src/api/app/policies/bs_request_policy.rb

```diff
@@ -2,7 +2,9 @@ class BsRequestPolicy < ApplicationPolicy
   def create?
     # new request should not have an id (BsRequest#number)
     return false if record.number
-    # dont let user set approver other than himself unless he is admin
-    ![nil, user.login].include?(record.approver) && !user.is_admin? ? false : true
+
+    return true if [nil, user.login].include?(record.approver) || user.is_admin?
```

Comment:
De Morgan would love this change.

Reply:
Well, its better than before. Isn't it? :+1: